Jenkinsfile

```diff
@@ -1,2 +1,29 @@
-// Build the plugin using https://github.com/jenkins-infra/pipeline-library
-buildPlugin(jenkinsVersions: [null, '2.107.1'])
+#!/usr/bin/env groovy
+
+@Library('sec_ci_libs@v2-latest') _
+
+def master_branches = ["master", "usi-jenkins", ] as String[]
+
+ansiColor('xterm') {
+  // using shakedown node because it's a lightweight alpine docker image instead of full VM
+  node('shakedown') {
+    stage("Verify author") {
+      user_is_authorized(master_branches, '8b793652-f26a-422f-a9ba-0d1e47eb9d89', '#marathon-dev')
+    }
+  }
+  node('mesos-med') {
```

Comment:
what kind of image/node is this? I don't see it here https://jenkins.mesosphere.com/service/jenkins/computer/

Reply:
It's the Mesos Medium for Docker in Docker